nexus/db-queries/src/db/datastore/switch_port.rs

```diff
@@ -359,6 +359,9 @@ impl DataStore {
                         crate::db::datastore::address_lot::try_reserve_block(
                             address_lot_id,
                             address.address.ip().into(),
+                            // TODO: Should we allow anycast addresses for switch_ports?
```

Comment:
Yes, we should. But we should require that this be an explicit aspect of an address that the user specifies, so a follow-up issue is fine here.

Reply:
https://github.com/oxidecomputer/omicron/issues/3628